packages/lesswrong/components/sunshineDashboard/SunshineNewUsersInfo.tsx

```diff
@@ -247,11 +256,10 @@ const SunshineNewUsersInfo = ({ user, classes, updateUser }: {
     limit: 50
   });
 
+  const commentKarmaPreviews = comments ? _.sortBy(comments, c => c[contentSort]) : []
+  const postKarmaPreviews = posts ? _.sortBy(posts, p=> p[contentSort]) : []
```

Comment:
If you just need to derefence an object property, `sortBy` will let you just pass in the property name as the second parameter like `_.sortBy(posts, contentSort)`.